Tests/NetworkServiceTests/InternalError/NetworkServiceInternalError+LocalizedErrorTests.swift

```diff
@@ -0,0 +1,12 @@
+import NetworkService
+import XCTest
+
+final class NetworkServiceInternalErrorLocalizedErrorTests: XCTestCase {
+    func testLocalizedDescriptionReturnExpectedString() {
+        // Given
+        let serializationFailDescription = "The provided data can not be serialized."
+
+        // When // Then
+        XCTAssertEqual(NetworkServiceInternalError.serializationFailure.errorDescription, serializationFailDescription)
```

Comment:
```suggestion
XCTAssertEqual(NetworkServiceInternalError.serializationFailure.errorDescription,
                       "The provided data can not be serialized.")
```
Can we make simpler?

Reply:
Fixed with https://github.com/ios-course/simple-network-service/pull/13/commits/92115fd5f550248944ac406b0dfded694803645a